google/datalab/contrib/bigquery/operators/_bq_execute_operator.py

```diff
@@ -31,13 +31,14 @@ def __init__(self, sql, parameters=None, table=None, mode=None, *args, **kwargs)
 
   def execute(self, context):
     query = bq.Query(sql=self._sql)
-    # allow_large_results is set to False because setting it True requires the destination table to
-    # be specified.
-    # use_cache is set to False since this is most likely the case for all pipeline scenarios
+    # allow_large_results is False because setting it True requires the destination table
+    # use_cache is False since this is most likely the case in pipeline scenarios
     output_options = bq.QueryOutput.table(name=self._table, mode=self._mode, use_cache=False,
                                           allow_large_results=False)
```

Comment:
Seems like in this case, you are actually producing results into a destination table, so allow_large_results should be True.

Reply:
Yep! Fixed, based on what the table name is.